### PR TITLE
chore(docs): update download version and file reference, update depen…

### PR DESCRIPTION
**Pull request by marcellobarile**
…dencies

Related to https://github.com/camunda/cawemo/issues/11725

### CODE REVIEW

**Review by wollefitz — CHANGES_REQUESTED** [review]

File: content/technical-guide/third-party-libraries/cawemo/cawemo-webapp.md
[the remark this answers is not in the document]
Comment (marcellobarile) [reply content/technical-guide/third-party-libraries/cawemo/cawemo-webapp.md:1]: sorry for that, I didnt remember that I had to run a fresh install before running that command. Now it should be fixed.

**Comment by marcellobarile** [comment]
Note: I'll upload the ZIP file as soon as the 1.9.16 image has been released

**Review by wollefitz — CHANGES_REQUESTED** [review]
@marcellobarile Both `static/download/cawemo-1.9.16.tar.gz` and `static/download/cawemo-1.9.16.zip` do **not** contain the correct files. You can get the correct archives by unzipping the artifact of https://github.com/camunda/cawemo/actions/runs/8601762685.

**Comment by marcellobarile** [comment]
> @marcellobarile Both `static/download/cawemo-1.9.16.tar.gz` and `static/download/cawemo-1.9.16.zip` do **not** contain the correct files. You can get the correct archives by unzipping the artifact of https://github.com/camunda/cawemo/actions/runs/8601762685.

Thanks for checking @wollefitz , I used the downloads from the `tags` list on GH. Just for my understanding, why they are not correct?

I'll add the ones from the action in the meantime.

**Comment by wollefitz** [comment]
> Got it why (probably); do the archives downloaded from the tags list contain the source while the ones from the action contain the built code?

Kind of. The assets attached to a tag contain the source code (this is also visible from the name of the assets: "Source Code"). These are automatically created by GitHub.

Screenshot 2024-04-09 at 10 25 24

The archives attached to the release job are created within the job, see https://github.com/camunda/cawemo/blob/0f4697220378a36708a407eb21ce0d78159f4151/.github/workflows/enterprise-release.yml#L157-L171.

They **do not** contain Cawemo binaries but the contents of the [`.enterprise-release` folder](https://github.com/camunda/cawemo/tree/master/.enterprise-release).

**Pull request merged**